tests/fixtures/slip10_ed25519.rs

```diff
@@ -30,11 +30,6 @@
                 chain_code: "8f6d87f93d750e0efccda017d662a1b31a266e4a6f5993b15f5c1f07f74dd5cc",
                 private_key: "30d1dc7e5fc04c31219ab25a27ae00b50f6fd66622f6e9c913253d6511d1e662",
             },
-            TestChain {
-                chain: Chain::from_u32_hardened(vec![0, 1, 2, 2]),
-                chain_code: "8f6d87f93d750e0efccda017d662a1b31a266e4a6f5993b15f5c1f07f74dd5cc",
-                private_key: "30d1dc7e5fc04c31219ab25a27ae00b50f6fd66622f6e9c913253d6511d1e662",
-            },
```

Comment:
Why is this removed?

Reply:
That was an exact duplicate of the test case above.